server/routes/apiRouter.js

```diff
@@ -27,8 +28,11 @@ const OrgInvoicingTypesTable = require("../models/OrgInvoicingTypesTable");
 const OrgFarmTypesTable = require("../models/OrgFarmTypesTable");
 const OrgAccountTypesTable = require("../models/OrgAccountTypesTable");
 
-apiRouter.get('/createDatabaseTables', (req, res) => {
-  if (reset === "yes") {
+apiRouter.get('/createDatabaseTables', passport.authenticate('jwt', { session: false }), (req, res) => {
```

Comment:
Calling `passport.authenticate` here requires that the user is logged in and uses the JWTStrategy middleware to load the data from the JWT into `req.user`.